src/containers/shared/components/Amount.tsx

```diff
@@ -6,11 +6,16 @@ import Currency from './Currency'
 import { ExplorerAmount } from '../types'
 
 export interface AmountProps {
-  value: ExplorerAmount
   displayIssuer?: boolean
+  modifier?: `+` | '-' | '~' // value to put in front of the currency symbol and number
+  value: ExplorerAmount
```

Comment:
nit: I think it's often good practice to put the optional parameters at the end, rather than the middle.